components/raft_log_engine/src/engine.rs

```diff
@@ -272,6 +276,7 @@ impl RaftEngine for RaftLogEngine {
     fn clean(
         &self,
         raft_group_id: u64,
+        _: u64,
```

Comment:
Raft engine should also avoid seeking, but I don't know how to do it either.

Reply:
The underlying implementation is just deleting a memtable and appending a tombstone, no seek involved. So no special care needed.